includes/templates/admin/dashboard.php

```diff
@@ -30,7 +30,11 @@
 }
 
 ?>
-
+<div id="stories-dashboard-no-js" class="wrap">
+	<div class="notice notice-error notice-alt">
+		<p><?php esc_html_e( 'The Web Stories dashboard requires JavaScript. Please enable JavaScript in your browser settings.', 'web-stories' ); ?></p>
```

Comment:
This way there's only 1 string to translate:

```suggestion
		<p><?php esc_html_e( 'Web Stories for WordPress requires JavaScript. Please enable JavaScript in your browser settings.', 'web-stories' ); ?></p>
```